truss/tests/util/test_basetenpointer.py

```diff
@@ -33,6 +34,10 @@ def test_dolly_12b():
                 "Skipping test due to ReadTimeout error from Hugging Face API, "
                 "this can happen for large models like Dolly-12b"
             )
+        except HfHubHTTPError as e:
```

Comment:
(optional): any easy way to introduce backoff here? rather than just skipping the suite